assets/dev/js/editor/document/hooks/ui/delete/move-resizeable-handle.js

```diff
@@ -0,0 +1,42 @@
+import After from 'elementor-api/modules/hooks/ui/after';
```

Comment:
It should be `assets/dev/js/editor/document/hooks/ui/document/elements/create/move-resizeable-handle.js`

Reply:
Done